lib/rules/block-closing-brace-newline-after/index.js

```diff
@@ -99,6 +99,22 @@ const rule = function(expectation, options) {
         index: -1,
         lineCheckStr: blockString(statement),
         err: msg => {
+          if (context.fix) {
+            if (expectation.indexOf("always") === 0) {
```

Comment:
Way not just use:
```js
if (expectation === "always") {
```

Reply:
@gucong3000 It matches `always`, `always-single-line`, `always-multi-line`.